packages/cadl-typescript/src/modelUtils.ts

```diff
@@ -536,6 +536,13 @@ function mapCadlTypeToTypeScript(
     case "ModelProperty":
       return mapCadlIntrinsicModelToTypeScript(program, cadlType, usage);
   }
+  if (cadlType.kind === undefined) {
```

Comment:
Does that mean the inputted `type` have no `kind` information from CADL?

If so it could be a bug from cadl, maybe we need to create bug from them to track this.

Reply:
I already mentioned this in the chat and Jose said he will confirm with cadl team.